website/src/components/Layout.tsx

```diff
@@ -40,7 +42,8 @@ export default class Layout extends React.PureComponent<LayoutProps> {
         return (
             <div className={`flex flex-column fill-height ${this.props.className || ''}`}>
                 <Helmet>
-                    <title>{metaProps.title}</title>
+                    {metaProps.seoTitle ? <title>{metaProps.seoTitle}</title> : <title>{metaProps.title}</title>}
```

Comment:
Can be simplified :)
```suggestion
                    <title>{metaProps.seoTitle || metaProps.title}</title>
```